tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -671,7 +676,11 @@ public function unlockItemAsPublicWithLastLockOfUserAndItemUsingWebDavAPI(
 		string $lockOwner,
 		string $itemToUseLockOf
 	) {
-		$user = $this->featureContext->getLastCreatedPublicShareToken();
+		if ($this->featureContext->isUsingSharingNG()) {
+			$user = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$user = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```

Comment:
```suggestion
		$token = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```
`$user` or `$token`?

Reply:
somewhere token is used and somewhere user was used in step definition. So I added whatever was in previous code